carbon/versions.json

```diff
@@ -559,6 +559,20 @@
         "linux/amd64": "https://github.com/Switcheo/carbon-bootstrap/releases/download/v2.62.0/carbond-mainnet.linux-amd64.tar.gz",
         "linux/arm64": "https://github.com/Switcheo/carbon-bootstrap/releases/download/v2.62.0/carbond-mainnet.linux-arm64.tar.gz"
       },
+      "next_version_name": "v2.63.0"
+    },
+    {
+      "name": "v2.63.0",
+      "proposal": 399,
+      "height": 69478121,
+      "recommended_version": "v2.63.0",
```

Comment:
just pointing out that the chain.json still indicated a recommended version of v2.53